src/lib/mongoBackend/mongoNotifyContextAvailability.cpp

```diff
@@ -37,12 +38,19 @@
 */
 HttpStatusCode mongoNotifyContextAvailability
 (
-  NotifyContextAvailabilityRequest*   requestP,
-  NotifyContextAvailabilityResponse*  responseP,
-  const std::string&                  tenant,
-  const std::string&                  servicePath
+  NotifyContextAvailabilityRequest*    requestP,
+  NotifyContextAvailabilityResponse*   responseP,
+  std::map<std::string, std::string>&  uriParam,
+  const std::string&                   tenant,
+  const std::string&                   servicePath
 )
 {
+    // FIXME P10: we have been discussing about changing the "notifyFormat" name for something more neutral and more
+    // meaningfull for *both* subscription notification and cpr forwards
+    std::string notifyFormat = uriParam[URI_PARAM_NOTIFY_FORMAT];
```

Comment:
As you can see, by the moment I'm using the existing "nofityFormat". However, this could be fixed in the next PR (yours) which finalize the feature in the upper layers.

NTC